rmw_dds_common/include/rmw_dds_common/context.hpp

```diff
@@ -27,16 +27,27 @@
 namespace rmw_dds_common
 {
 
+/// Structure of data that a Context will need in any rmw implementation
+/// mapping one Participant to Multiple Nodes.
 struct Context
 {
+  /// Gid of the Participant that the context uses.
   rmw_gid_t gid;
+  /// Publisher used to publish ParticipantEntitiesInfo discovery data.
   rmw_publisher_t * pub;
+  /// Subscriber used to listen ParticipantEntitiesInfo discovery data.
   rmw_subscription_t * sub;
+  /// Cached graph from listened discovery data.
```

Comment:
@ahcorde @ivanpauno nit:
```suggestion
  /// Cached graph from discovery data.
```
as it includes in-process data, no?